### ISSUE TITLE
overload issues at dl.bintray.com repo

### ISSUE THREAD

**Issue by kelsieflynn** [bennettpeter/mythtv#4]
Since I've respun several versions of raspbian based off RPI-Distro/pi-gen lately and your mythtv-light, I used quite a bit of bandwidth over at your repo, which I apologize for, if it caused any problems.
Anyway, in the future I will take snapshots of your repos to use for my compilations to avoid excess bandwidth. I've already started  for myth28 and will migrate other versions into it as well.

It's over at:

````
deb https://sourceforge.net/projects/mpbrepo/files/20180502/deb/ stretch myth28
````

*I also use apt-cacher-ng now as well on most rebuilds, which seems to help a lot.

Thanks for all your hard work and efforts in  making mythtv the best I've seen in a while.

 Best Regards.

**Comment by bennettpeter** [comment]
No problem, I am sure they have plenty of bandwidth. I checked and there is not a noticable increase. There is a spike in downloads the day after I release a new build, more than anything you may have done.

If you want to save on downloads you can just get the deb file (e.g. mythtv-light_29.1-14-g56030f3f2a-0_armhf_stretch.deb) and install it on raspbian with gdebi.

**Comment by kelsieflynn** [comment]
Ok, good. I wouldnt have mentioned it except I received several 403 error codes after accessing the repo a lot over the last two weeks, which you might see , if you see excessive use from a ip via Ateetee-YouVerse, Ohio origin, with ip ending in forty8.dot.elev3n, that would have been me.
Good to know nothing I did likely affected others access They were likely just limited me temporarily or It could have been on my end, but I dont think so.

 I havent used gdebi before, in the old days I would have to specify all deps manually with a manual download or download them all together, then install using dpkg/apt. It's been so long, I can imagine it(gdebi) makes it easier to do now. I'm  not sure about scripting it into the projects for building raspbian images. I have a fair generic understanding, but I still have a lot of details to understand with all the options of debootstrap, apt-secure  and all with regard to raspbian image building.. Eg. debootstrap has a tarball option, that would be perfect for my current needs, I tried it twice, but I have yet to make it work properly with my forked pi-gen build system, I need to experiment on it more.
Once I do that though, i should be able to pull all debs from all repos in a session, tarred but with the chroot step suspended,  and then repeat installs/starting from the chroot tarball. Very powerful options at a low level, just like I enjoy.

Thank you ,  close this ticket at your leisure.
